docs/sql-migration-guide.md

```diff
@@ -32,6 +32,8 @@ license: |
 
   - In Spark 3.2, script transform default FIELD DELIMIT is `\u0001` for no serde mode. In Spark 3.1 or earlier, the default FIELD DELIMIT is `\t`.
 
+  - In Spark 3.2, script transform default serde property `field.delim` is `\t` for Hive serde mode when user specifies serde. In Spark 3.1 or earlier, the default serde property `field.delim` is `\u0001` for Hive serde mode when user specifies serde.
```

Comment:
@AngersZhuuuu can you combine with the item above and clarify it together?

Reply:
> @AngersZhuuuu can you combine with the item above and clarify it together?

Updated.